core/src/main/resources/db/changelog/changesets/2024_09_18_survey_assign_rename.yaml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
+  - changeSet:
+      id: "survey_assign_rename"
+      author: dbush
+      changes:
+        - addColumn:
+            tableName: survey
+            columns:
+              - column: { name: auto_assign, type: boolean, defaultValueBoolean: true, constraints: { nullable: false } }
```

Comment:
to avoid downtime, this is operating as a create new, and migrate.  In the next release, we'll delete the old column